astro/src/content/blog/understanding-multifactor-authentication-mfa.mdx

```diff
@@ -0,0 +1,64 @@
+---
+
+Picture a timeline where the starting point is the very first person entering a username and password combination. As the line moves on, you see how the world of user login has changed. Then there's a big "You Are Here" arrow. It points to an area labeled [multi-factor authentication](/articles/authentication/multi-factor-authentication).
+
+Now yes, you could argue that we're well beyond MFA already. To some extent, you'd be right. But it's also accurate to say that MFA covers a wide swath of the timeline, and we're still inside of its boundaries today. While we are moving toward a future where [passwords go away](/articles/authentication/passwordless-authentication-security), we're still working to get MFA done right. Adoption is still [under 40 percent](https://www.resmo.com/blog/multifactor-authentication-statistics) in small businesses; a worrying statistic when you also grasp that nearly 60 percent of people still write down passwords on sticky notes. Of those, almost 70 percent [admit](https://www.lastpass.com/state-of-the-password/global-password-security-report-2019) to losing the notes at some point.
+
+As MFA solutions are enjoying their peak, it's a good time to dive in and get a better understanding of the landscape we're in today.
+
+## Understanding Multi-Factor Authentication
+
+Multi-factor authentication (MFA) does what it says on the box. It requires users to provide two or more verification "factors" to gain access to a resource. To be clear, two-factor authentication is also a form of multi-factor authentication. In fact, it's the most common.
+
+Anyway, those factors may be an application, an online account, or a service like a VPN. Traditional authentication methods rely on a single factor--something you know (like a password.) MFA adds extra layers of security by requiring something you have (like an authenticator app or SMS with a one-time password on a mobile device), or something you are (like [a fingerprint](/blog/introducing-biometric-authentication)) to help keep cybercriminals, hackers, and data breaches at bay.
+
+This second factor and third factor make MFA a strong barrier against bad actors seeking unauthorized access. We live in a time where each week seems to bring [another news story](/blog/grammarly-proves-ciam-not-optional) of a security breach. If today is the most important time to have better security, tomorrow will be even more important.
+
+## The Importance of MFA in Cybersecurity
+
+It would be hard to overstate how important MFA is. As we mentioned earlier, we see new threats emerging all the time. These threats [grow more sophisticated](blog/fusionauth-releases-advanced-threat-detection), and the limitations of simple password protection becomes glaringly evident. While passwords were once considered to be the highest form of security, they're now easily compromised. Phishing attacks, social engineering, and brute force methods are like a wrecking ball against a glass house.
```

Comment:
Some implementations of passwords can be easily compromised, correct implementations and strong passwords are still not a trivial thing to break.  You may want to say something like

  ```suggestion
It would be hard to overstate how important MFA is. As we mentioned earlier, we see new threats emerging all the time. These threats [grow more sophisticated](blog/fusionauth-releases-advanced-threat-detection), and the limitations of simple password protection becomes glaringly evident. While passwords were once considered to be the highest form of security, they're now susceptible to common vulnerabilities. Phishing attacks, social engineering, and brute force methods are like a wrecking ball against a glass house.
```

If you wanted to add humor you could say something in addition like: Not to mention the sticky note issue, which is like bird poop on your jacket. (or some other funny analogy.)